src/services/session/SessionService.js

```diff
@@ -405,7 +405,7 @@ class SessionService {
     let notificationString = 'Historical queries not restricted by session.';
 
     if (model) {
-      notificationString = `Historical queries restricted to ${model.numbers.length} ${model.numbers.length > 1 ? "session's" : 'session'} on ${model.host}.`;
+      notificationString = `Historical queries restricted to ${model.numbers.length} ${model.numbers.length > 1 ? "session's" : 'session'} (${formatMultipleSessionNumbers(model.numbers)}) on ${model.host}.`;
```

Comment:
suggest replacing ` on ${model.host}.` with `${model.host !== undefined ? ` on ${model.host}` : ''}`. Oof, nested template literals... first time for everything.

Reply:
donzo